cmake/UserOverride.cmake

```diff
@@ -8,7 +8,7 @@
 
 if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     # g++
-    set(common "-std=c++11 -Wall -Wextra -fno-common")
+    set(common "-Wall -Wextra -Wno-unused-parameter -fno-common")
     if (NOT MINGW)
         set(common "${common} -fPIC")
```

Comment:
Position independent code flags should go into CMakeLists.txt as well.

Reply:
Fixed in 100a52e07813ea54c7b3282adf733861dd09e27a.